pom.xml

```diff
@@ -24,7 +24,7 @@
     <maven.nexus-staging.plugin.version>1.6.13</maven.nexus-staging.plugin.version>
 
     <lombok.version>1.18.30</lombok.version>
-    <citrus.version>4.2.1</citrus.version>
+    <citrus.version>4.3.0-SNAPSHOT</citrus.version>
```

Comment:
well, this will not build. but I think it's dependent on https://github.com/citrusframework/citrus/pull/1177, right?